### DISCUSSION TITLE
Trying to run the demos

### DISCUSSION THREAD

**Discussion by oshimoto** [ebitenui/ebitenui#82] [Q&A] [answered]
Hi! I downloaded the examples/demo folder and when running "go mod tidy" I get the following error:

go: found golang.org/x/text/language in golang.org/x/text v0.11.0
dnd imports
        github.com/hajimehoshi/ebiten/v2 imports
        github.com/hajimehoshi/ebiten/v2/internal/ui imports
        golang.org/x/mobile/app imports
        golang.org/x/exp/shiny/driver/gldriver: ambiguous import: found package golang.org/x/exp/shiny/driver/gldriver in multiple modules:
        golang.org/x/exp v0.0.0-20190731235908-ec7cb31e5a56 (C:\go-apps\pkg\mod\golang.org\x\exp@v0.0.0-20190731235908-ec7cb31e5a56\shiny\driver\gldriver)
        golang.org/x/exp/shiny v0.0.0-20230522175609-2e198f4a06a1 (C:\go-apps\pkg\mod\golang.org\x\exp\shiny@v0.0.0-20230522175609-2e198f4a06a1\driver\gldriver)
dnd imports
        github.com/hajimehoshi/ebiten/v2 imports
        github.com/hajimehoshi/ebiten/v2/internal/ui imports
        golang.org/x/mobile/app imports
        golang.org/x/exp/shiny/screen: ambiguous import: found package golang.org/x/exp/shiny/screen in multiple modules:
        golang.org/x/exp v0.0.0-20190731235908-ec7cb31e5a56 (C:\go-apps\pkg\mod\golang.org\x\exp@v0.0.0-20190731235908-ec7cb31e5a56\shiny\screen)
        golang.org/x/exp/shiny v0.0.0-20230522175609-2e198f4a06a1 (C:\go-apps\pkg\mod\golang.org\x\exp\shiny@v0.0.0-20230522175609-2e198f4a06a1\screen)

What am I doing wrong? Please help me run the demo!

**Comment by mcarpenter622** [comment, marked as answer]
You have to run them from the root directory. See below. This will be added
to the readme soon.

* Ebiten UI complete demo:
 go run github.com/ebitenui/ebitenui/_examples/demo

* Ebiten UI widget:
go run github.com/ebitenui/ebitenui/_examples/widget_demos/<folder_name>

**Reply by oshimoto** [reply]
Ok thanks a lot! Will try to run them as suggested.
